src/app/about-us/members/members.service.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { AngularFirestore } from '@angular/fire/firestore';
+
+import { Members } from '../../shared/models/members.model';
+import { DataService } from '../../shared/models/data-service.model';
+import { FirebaseCollections } from '../../shared/models/collections.constant';
+
+@Injectable({
+  providedIn: 'root'
```

Comment:
we should remove this as it is already provided by the module.

Reply:
Fixed